docs/tutorials/configure/pet-photographer.md

```diff
@@ -23,7 +23,7 @@ After following this tutorial, you will understand how to control sync parameter
 Note: Consider this tutorial alongside filtered camera tutorial.
 -->
 
-If your smart machine [captures](/services/data/capture/) a lot of data, you might want to filter captured data to selectively store only the data you are interested in.
+If your smart machine [captures](/services/data/capture-sync/) a lot of data, you might want to filter captured data to selectively store only the data you are interested in.
```

Comment:
just my personal opinion and not sure what the docs-wide stance is on this, but imo "smart machine" is a marketing term and i find it strange in the docs.  i think "machine" makes more sense. i think "smart machine" makes technical ppl feel like they're being sold to when they just want to get dry facts.

Reply:
i know change here was to the linked URL so feel free to ignore lol; just wanted to mention

Reply:
```suggestion
If your machine [captures](/services/data/capture-sync/) a lot of data, you might want to filter captured data to selectively store only the data you are interested in.
```

Reply:
```suggestion
If your machine [captures](/services/data/capture-sync/) a lot of data, you might want to filter captured data to selectively store only the data you are interested in.
```